supplementary_simulations/direct_classification_with_RESTNET18.ipynb

```diff
@@ -0,0 +1,386 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Direct classification with RESENT18 (Example 2: *ants* / *bees*)"
```

Comment:
```suggestion
    "# Direct classification with RESNET18 (Example 2: *ants* / *bees*)"
```
Although I do like the sound of the resent network!

Note also that the filename is "RESTNET", i.e., with an extra T.